<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingReview</class>
 <widget class="QWidget" name="BookingReview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>实验室预约审核系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="filterGroupBox">
     <property name="title">
      <string>筛选条件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>预约状态:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="statusComboBox">
        <item>
         <property name="text">
          <string>全部</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>待审核</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>已通过</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>已拒绝</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="dateLabel">
        <property name="text">
         <string>预约日期:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDateEdit" name="dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="keywordLabel">
        <property name="text">
         <string>搜索关键字:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="keywordLineEdit"/>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="searchBtn">
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="refreshBtn">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="bookingTableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="approveBtn">
       <property name="text">
        <string>通过审批</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rejectBtn">
       <property name="text">
        <string>拒绝申请</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="importBtn">
       <property name="text">
        <string>导入数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportBtn">
       <property name="text">
        <string>导出数据</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
